include/hpdf.h

```diff
@@ -789,6 +789,10 @@ HPDF_EXPORT(HPDF_Image)
 HPDF_LoadJpegImageFromFile (HPDF_Doc      pdf,
                             const char    *filename);
 
+HPDF_EXPORT(HPDF_Image)
+HPDF_LoadJpegImageFromFile2 (HPDF_Doc      pdf,
```

Comment:
Shouldn't we find a better name such as `LazyLoadJpegImageFromFile` or `LoadJpegImageFromFileDelayed`?

Reply:
I chose to stick to the same name as Png for consistency…